qemu/tests/block_stream_stress.py

```diff
@@ -1,100 +1,59 @@
-import time
 import logging
 
-from autotest.client.shared import error
-from autotest.client import utils
+from avocado.core import exceptions
```

Comment:
As https://github.com/avocado-framework/avocado-vt/pull/941#issuecomment-288009679 mentioned, better to use `test` instance to report the status.

http://avocado-framework.readthedocs.io/en/latest/api/test/avocado.html